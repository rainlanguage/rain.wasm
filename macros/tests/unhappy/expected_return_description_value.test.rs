#[macro_use]
extern crate wasm_bindgen_utils_macros;

struct TestStruct;

#[wasm_export]
impl TestStruct {
    #[wasm_export(return_description)]
    pub async fn some_method_missing_return_desc_value(arg: String) -> Result<String, Error> {
        Ok(String::new())
    }
}

fn main() {}