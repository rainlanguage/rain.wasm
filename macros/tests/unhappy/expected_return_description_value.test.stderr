error: expected a value for this attribute: `return_description = ...` and it must be a string literal
 --> tests/unhappy/expected_return_description_value.test.rs:8:19
  |
8 |     #[wasm_export(return_description)]
  |                   ^^^^^^^^^^^^^^^^^^
